modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add Network Interface*.
+. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. Click *Add Network Interface* to add the network interface without restarting the virtual
```

Comment:
I'm having issues in opening the web console, so I can't see or try it for myself, but this section seems weird: The user already clicked "Add Network Interface" 2 sections ago, and in the previous section he was still in the "Add Network Interface" window, so it's seems odd he should click "Add Network Interface" again.
Please re-check.

Reply:
on 4.3 it is:
On the nic list there is an "Add Network Interface" button to open the add nic dialog. After you fill in all the details, you need to submit the dialog; that submit button is called "Add"

on 4.2 it is:
On the nic list there is an "Create Nic" button. If you click it, not a dialog but an editable row will show up. To submit it, you need to press the check-mark (:heavy_check_mark:) button on the right hand side of the row.

Reply:
For 4.2 changed these steps to:

. From the *{object-gui}* tab, select the {object}.
. Select the *Network Interfaces* tab.
. Click *Create Network Interface*.
. In the *Create Network Interface* list row, specify the
*Name*, *Model*, *Network*, *Type*,
and *MAC Address* of the network interface.
. Click the green check-mark to the right of the row to add the network interface.
. Restart the virtual machine to enable access.
. Edit the drop-down lists and check boxes to configure the network
interface.
. Click *Save Changes*.
. Click *OK*.